.gitmodules

```diff
@@ -217,3 +217,6 @@
 [submodule "projects/braincode_POND_Registry_Imaging_Data_Release"]
 	path = projects/braincode_POND_Registry_Imaging_Data_Release
 	url = https://github.com/conpdatasets/braincode_POND_Registry_Imaging_Data_Release
+[submodule "projects/C-OPN_LORIS"]
+	path = projects/C-OPN_LORIS
+	url = https://github.com/conpdatasets/C-OPN_LORIS.git
```

Comment:
This description is fine and about 2/3 of was used for the description field in the DATS.json.  We need to check in with Samir and JB for a realistic timeline of data release here so we know if we have anything to point to.  Depending on that timeline, it may also be a candidate for a 'coming soon' banner if we implement that feature.